### ISSUE TITLE
update 2 photon session schema

### ISSUE THREAD

**Issue by DowntonCrabby** [AllenNeuralDynamics/aind-data-schema#253]
Updating the 2-photon session schema. 

Updated Schema will roughly follow this outline: 

- Ophys_Session_type: (str) "2-Photon"
-  Metadata: (object)
    - Session id
    - Mouse id
    - Rig id
    - User id

-  Subject (object):
    - nested subject schema 

- ophys_rig (object):
    - nested ophys_rig schema

- Data streams 
    - eye tracking video
    - body tracking video
    - face tracking video
    - wheel encoder
    - stimulus
    - physio
        - field of view: 
            - nested field of view schema 
     - behavior task
         - lick detection
         - water/rewards delivery
         - task parameters
         - task performance

**Comment by saskiad** [comment]
Can we break these into smaller pieces? I think it'll be easier to get them aligned.

eg. Several of these data streams are already modeled for the ephys session and we can bring those over. It's very important to us to keep our schemas for different modalities as closely aligned as possible. (so far our ophys users haven't needed them, which is why they aren't here yet)

**Comment by DowntonCrabby** [comment]
yes, I was thinking that we would be making a schema for each of the datastreams

**Comment by DowntonCrabby** [comment]
@saskiad can you please point me to where the streams are already modeled? I've been looking here: 
https://github.com/AllenNeuralDynamics/aind-data-schema/blob/main/src/aind_data_schema/ephys/ephys_session.py

From what I can tell you're referring to [this stream model](https://github.com/AllenNeuralDynamics/aind-data-schema/blob/2d18f1657f4faf747af26e9da71fdcbc45c35b4f/src/aind_data_schema/ephys/ephys_session.py#L87) and [implemented in the session schema here](https://github.com/AllenNeuralDynamics/aind-data-schema/blob/2d18f1657f4faf747af26e9da71fdcbc45c35b4f/src/aind_data_schema/ephys/ephys_session.py#L115)

These look a little more generic but if there is something more specific I'd love to take a look

**Comment by saskiad** [comment]
Many of these are modeled in the device schema:
https://github.com/AllenNeuralDynamics/aind-data-schema/blob/main/src/aind_data_schema/device.py

**Comment by saskiad** [comment]
closed with PR #266
